FusionIIIT/applications/academic_information/models.py

```diff
@@ -146,15 +164,15 @@ def __self__(self):
 
 class Timetable(models.Model):
     upload_date = models.DateTimeField(auto_now_add=True)
-    time_table = models.CharField(max_length=20)
+    time_table = models.FileField(upload_to='documents/%Y/%m/%d')
 
     class Meta:
         db_table = 'Timetable'
 
 
 class Exam_timetable(models.Model):
     upload_date = models.DateField(auto_now_add=True)
-    exam_time_table = models.CharField(max_length=20)
+    exam_time_table = models.FileField(upload_to='documents/%Y/%m/%d')
```

Comment:
@3Peers/academic-information-management, Look at this ==> https://github.com/3Peers/FusionIIIT/issues/90